basic_example_camera:
  ros__parameters:
    image_topic: "camera/image_mono"
    latency_topic: "camera/latency"
    add_timestamp: True
    publish_grayscale: True
    publish_color: False
    latch_timing_interval_s: 5
    reset_camera_settings: False
    camera_settings:
      TriggerMode: "Off"
      GainAuto: "Continuous"
      AcquisitionFrameRateAuto: "Off"
      AcquisitionFrameRateEnabled: True
      AcquisitionFrameRate: 50
      BinningVertical: 2
      unknown_param: "test, test"
    camera_chunkdata:
      FrameCounter:
        ChunkEnable: True
        ChunkModeActive: True
      randomChunk:
        blabla: False
      Timestamp:
        ChunkEnable: True
        ChunkModeActive: True
camera_test_off:
  ros__parameters:
    image_topic: "camera/image_mono"
    latency_topic: "camera/latency"
    camera_settings:
      TriggerMode: Off
      GainAuto: Off
      Gain: 18
      AcquisitionFrameRateAuto: Off
      AcquisitionFrameRateEnabled: True
      AcquisitionFrameRate: 100
camera_test_id:
  ros__parameters:
    image_topic: "camera/image_mono"
    latency_topic: "camera/latency"
    cam_id: 18518215
    publish_grayscale: True
    publish_color: True
